modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -10943,6 +10945,7 @@ move-selected-item-in-x-one-position-up=Move selected item in {0} one position u
 move-selected-items-from-x-to-x=Move selected items from {0} to {1}.
 move-thread=Move Thread
 move-to=Move To
+move-to-current-folder=Move To Current Folder
```

Comment:
@igor-franca following https://capitalizemytitle.com/, this should be "Move to Current Folder"